src/index.css

```diff
@@ -0,0 +1,4 @@
+@import "./styles/reset.css";
+@import "./styles/stars.css";
+@import "./styles/card.css";
+@import "./styles/variables.css";
```

Comment:
Just a tip - you can add comments to describe the files which you imported. For example, before first import add comment /* RESET SETTINGS */  etc. If you don't agree - you don't have to add comments like this.

Reply:
Great tip, thank you, I'll remember it